src/main/java/Task.java

```diff
@@ -0,0 +1,25 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
```

Comment:
Good job naming your boolean variable to sound like a boolean as per the coding standards.